src/frontend/efiling-frontend/src/components/page/payment/Payment.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { MdCreditCard } from "react-icons/md";
+import ConfirmationPopup, {
+  Button,
+  Sidecard,
+  Alert,
+  DisplayBox,
+  Table
+} from "shared-components";
+import { getSidecardData } from "../../../modules/sidecardData";
+import { propTypes } from "../../../types/propTypes";
+
+// TODO: Extract to module to generate this
+const feesData = [
+  { name: "Some Fees:", value: "$100.00", isValueBold: true },
+  { name: "Some More Fees:", value: "$10.00", isValueBold: true },
+  { name: "", value: "", isEmptyRow: true },
+  { name: "Total Fees:", value: "$110.00", isValueBold: true }
```

Comment:
Same as above, something pulled from API? If no endpoint exists, could be in a testdata module?

Reply:
same